service-front/module/Application/view/application/authenticated/lpa/reuse-details/index.twig

```diff
@@ -1,4 +1,4 @@
-{% extends isPopup ? 'layout/twig/popup.twig' : 'layout/twig/layout.twig' %}
+{% extends isPopup ? 'layout/popup.twig' : 'layout/twig/layout.twig' %}
```

Comment:
noticed that the `layout.twig` here is in a different place to one i'm marking below - is this right?

Reply:
there are several of these so just checking

Reply:
Thanks for spotting these @williamfalconeruk I've now amended. Interesting that UAT still passed and the functional tests... Though this may only affect layout in very particular circumstances which aren't picked up by either.